src/main/programming/logic_condition.h

```diff
@@ -47,29 +47,30 @@ typedef enum {
     LOGIC_CONDITION_SUB                         = 15,
     LOGIC_CONDITION_MUL                         = 16,
     LOGIC_CONDITION_DIV                         = 17,
-    LOGIC_CONDITION_GVAR_SET                    = 18,
-    LOGIC_CONDITION_GVAR_INC                    = 19,
-    LOGIC_CONDITION_GVAR_DEC                    = 20,
-    LOGIC_CONDITION_PORT_SET                    = 21,
-    LOGIC_CONDITION_OVERRIDE_ARMING_SAFETY      = 22,
-    LOGIC_CONDITION_OVERRIDE_THROTTLE_SCALE     = 23,
-    LOGIC_CONDITION_SWAP_ROLL_YAW               = 24,
-    LOGIC_CONDITION_SET_VTX_POWER_LEVEL         = 25,
-    LOGIC_CONDITION_INVERT_ROLL                 = 26,
-    LOGIC_CONDITION_INVERT_PITCH                = 27,
-    LOGIC_CONDITION_INVERT_YAW                  = 28,
-    LOGIC_CONDITION_OVERRIDE_THROTTLE           = 29,
-    LOGIC_CONDITION_SET_VTX_BAND                = 30,
-    LOGIC_CONDITION_SET_VTX_CHANNEL             = 31,
-    LOGIC_CONDITION_SET_OSD_LAYOUT              = 32,
-    LOGIC_CONDITION_SIN                         = 33,
-    LOGIC_CONDITION_COS                         = 34,
-    LOGIC_CONDITION_TAN                         = 35,
-    LOGIC_CONDITION_MAP_INPUT                   = 36,
-    LOGIC_CONDITION_MAP_OUTPUT                  = 37,
-    LOGIC_CONDITION_RC_CHANNEL_OVERRIDE         = 38,
-    LOGIC_CONDITION_SET_HEADING_TARGET          = 39,
-    LOGIC_CONDITION_LAST                        = 40,
+    LOGIC_CONDITION_MODULUS                     = 18,
```

Comment:
Please add it a LAST entry not to break compatibility with existing solutions

Reply:
I think there also numbers of conditions should be preserved, so the Modulus is number 40, and the LAST goes to 41.

Reply:
I moved it to the last entry, one of the commit i referred to moved the enum number around so i just followed suit.